python/feldera/_callback_runner.py

```diff
@@ -37,6 +38,19 @@ def run(self):
         :meta private:
         """
 
+        pipeline = self.client.get_pipeline(self.pipeline_name)
```

Comment:
Schema doesn't change after the pipeline is compiled, so it should be possible to cache this information once per pipeline.